benchto-driver/src/test/java/io/trino/benchto/driver/utils/PermutationUtilsTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.benchto.driver.utils;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.stream.IntStream;
+
+import static org.junit.Assert.assertArrayEquals;
```

Comment:
is there `assertArrayNotEquals`?

Reply:
nope...

Reply:
never mind, will use assertThat() anyway